### PR TITLE
Update/name field

### PR DESCRIPTION
**Pull request by nickbristow**
As per https://www.pivotaltracker.com/story/show/93638458

this PR requires a DB migration

c2

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -65,6 +65,28 @@
...
+    t.text     "notes"
+    t.integer  "quantity"
+    t.text     "details"
+    t.string   "part_number"
+    t.float    "price"
+    t.integer  "cart_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
```
Comment (afeld) [inline db/schema.rb]: Not sure what happened here...I can fix it if you need.

Comment (afeld) [inline db/schema.rb]: Your DB might be in a weird state.

File: app/models/ncr/work_order.rb [hunk 2]
```diff
@@ -110,6 +110,9 @@ def version
       self.updated_at.to_i
     end
 
+    def name
+      self.project_title
+    end
```
Comment (afeld) [inline app/models/ncr/work_order.rb]: Can we get rid of this?

**Comment by afeld** [comment]
Nice work! Just a couple minor things.

File: app/models/ncr/work_order.rb (hunk 2 above)
Comment (cmc333333) [inline app/models/ncr/work_order.rb]: Not quite -- `name` is part of the `client_data` interface; it's used in most listing and in a few email templates, iirc.

Comment (afeld) [inline app/models/ncr/work_order.rb]: Ah ok, no worries then.

**Pull request merged**